lib/ansible/modules/clustering/kubevirt/kubevirt_vm.py

```diff
@@ -0,0 +1,312 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2018, KubeVirt Team <@kubevirt>
+# Apache License, Version 2.0
+# (see LICENSE or http://www.apache.org/licenses/LICENSE-2.0)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: kubevirt_vm
+
+short_description: Manage KubeVirt virtual machine
+
+description:
+    - Use Openshift Python SDK to manage the state of KubeVirt virtual machines.
+
+version_added: "2.8"
+
+author: KubeVirt Team (@kubevirt)
+
+options:
+    state:
+        description:
+            - Set the virtual machine to either C(present), C(absent), C(running) or C(stopped).
+        default: "present"
+        choices:
+            - present
+            - absent
+            - running
+            - stopped
+        type: str
+    name:
+        description:
+            - Name of the virtual machine.
+        required: true
+        type: str
+    namespace:
+        description:
+            - Namespace where the virtual machine exists.
+        required: true
+        type: str
+    disks:
+        description:
+            - List of dictionaries which specify disks of the virtual machine.
+            - A disk can be made accessible via four different types: I(disk), I(lun), I(cdrom), I(floppy).
```

Comment:
As far as I can see we do not provide such information when defining disks. Please see [here](https://github.com/kubevirt/kubevirt/blob/master/cluster/examples/vmi-fedora.yaml#L11).

Reply:
Not sure I follow. You mean that disk type don't have to be specified in case we want to remove disk?

Reply:
I meant that we do not specify things like luns, cdroms etc.